src/validation.h

```diff
@@ -206,19 +206,23 @@ struct PackageMempoolAcceptResult
 };
 
 /**
- * Try to add a transaction to the mempool. This is an internal function and is
- * exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
+ * Try to add a transaction to the mempool with a specified acceptance time. This is an internal
+ * function and is exposed only for testing. Client code should use ChainstateManager::ProcessTransaction()
  *
- * @param[in]  active_chainstate  Reference to the active chainstate.
+ * @param[in]  chainparams        Tweakable parameters of this instance.
  * @param[in]  pool               Reference to the node's mempool.
+ * @param[in]  active_chainstate  Reference to the active chainstate.
  * @param[in]  tx                 The transaction to submit for mempool acceptance.
+ * @param[in]  nAcceptTime        The acceptance time.
  * @param[in]  bypass_limits      When true, don't enforce mempool fee and capacity limits.
  * @param[in]  test_accept        When true, run validation checks but don't submit to mempool.
  *
  * @returns a MempoolAcceptResult indicating whether the transaction was accepted/rejected with reason.
  */
-MempoolAcceptResult AcceptToMemoryPool(CChainState& active_chainstate, CTxMemPool& pool, const CTransactionRef& tx,
-                                       bool bypass_limits, bool test_accept=false) EXCLUSIVE_LOCKS_REQUIRED(cs_main);
+MempoolAcceptResult AcceptToMemoryPool(const CChainParams& chainparams, CTxMemPool& pool,
+                                                        CChainState& active_chainstate, const CTransactionRef &tx,
+                                                        int64_t nAcceptTime, bool bypass_limits, bool test_accept)
```

Comment:
prefer snake case `accept_time`. Doxygen comment should be more descriptice, e.g. that this is the time that the mempool entry will record and will be used to determine when the entry expires.

Reply:
Changed `nAcceptTime` to `accept_time` and improved the field description in 034e827